jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/DateTimePickers.java

```diff
@@ -2,9 +2,53 @@
 
 import com.epam.jdi.light.common.JDIAction;
 import com.epam.jdi.light.elements.base.UIBaseElement;
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.complex.ISetup;
+import com.epam.jdi.light.material.annotations.JDIPicker;
 import com.epam.jdi.light.material.asserts.inputs.DateTimePickersAssert;
 
-public class DateTimePickers  extends UIBaseElement<DateTimePickersAssert> {
+import java.lang.reflect.Field;
+
+import static com.epam.jdi.light.elements.pageobjects.annotations.objects.FillFromAnnotationRules.fieldHasAnnotation;
+
+public class DateTimePickers extends UIBaseElement<DateTimePickersAssert> implements ISetup {
+
+    private String pickerInput;
+    private String pickerLabel;
+    private String pickerBtn;
+
+
+    @Override
+    public void setup(Field field) {
+        if (!fieldHasAnnotation(field, JDIPicker.class, DateTimePickers.class))
+            return;
+        JDIPicker annotation = field.getAnnotation(JDIPicker.class);
+
+        pickerInput = annotation.pickerInput();
+        pickerLabel = annotation.pickerLabel();
+        pickerBtn = annotation.pickerButton();
+    }
+
+    @Override
+    public DateTimePickersAssert is() {
+        return new DateTimePickersAssert().set(this);
+    }
+
+    @JDIAction("Get picker field with index {0}")
+    public UIElement getPickerField(int index){
```

Comment:
Please use DateTimePickers instead of UIElement as it's done for Button. Possible you'll need do some other updates because o it